analyses/infercnv-consensus-cell-type/references/normal-references/README.md

```diff
@@ -5,3 +5,6 @@ The RDS files are not stored in version control due to their large size.
 Each project directory will also have a TSV tile listing all of the consensus cell types included in each reference, which is included in version control.
 
 All files are created by project-specific scripts found in `../../scripts/build-normal-reference/`.
+
+The TSV file `reference-cell-groups.tsv` contains consensus cell types organized into reference cell group categories, and it is used to help create references across projects.
```

Comment:
I think I would move this file to be in `infercnv-consensus-cell-type/references` alongside the CNV reference file and keep the `normal-references` folder for storing the actual references themselves? I don't feel strongly, it's just a lot of nested directories and I think we could decrease that.